### PR TITLE
(selenium-chromium-edge-driver) Get stable driver version directly from MS

### LINKED ISSUE #2241: (selenium-chromium-edge-driver) Outdated
### Checklist

- [X] I have checked the moderation queue that no new version has been submitted (only visible when logged in), or the submitted package version has failed the automated checks.

Please navigate to the following [link](https://community.chocolatey.org/packages?q=&moderatorQueue=true&moderationStatus=all-statuses&prerelease=false&sortOrder=package-download-count) to view the moderation queue.

- [X] I have verified that the new version is a stable release.
- [X] I have looked to see if there are any Bug reports that prevents a new version being submitted and that no existing Outdated Reports have been created.
- [X] I have verified the new version has a Windows binary (EXE or MSI installer, Zip or other archive, etc).

### New Software Version

115.0.1860.0:

### Download location

https://developer.microsoft.com/en-us/microsoft-edge/tools/webdriver/

### Package Page

https://community.chocolatey.org/packages/selenium-chromium-edge-driver

### PR DESCRIPTION
**Pull request by patest-dev**
## Description

Changed the automatic update script to fetch the latest stable driver version directly from MS.

## Motivation and Context

Currently the package is only updated if the edge driver download page has a driver link with the version matching the current version of the microsoft-edge package.

The versions of the selenium driver and browser don't necessarily match up and this caused the update script to be broken for quite a long time. Because of this now quite a few versions of the selenium drivers are missing on chocolatey.

Loading the latest stable driver version from MS and using that for the download links is a more stable approach for the update script. 

Fixes #2241 

## How Has this Been Tested?

Run locally
Had the almost identical PS script in use for the last ~ 1.5 years because for quite some time there was no up-to-date edge driver package.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:
- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [ ] I have updated the package description and it is less than 4000 characters.
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [ ] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

**Comment by patest-dev** [comment]
@AdmiringWorm @gep13 @NotMyFault @JPRuskin Any chance of reviewing this?

**Comment by pauby** [comment]
@patest-dev please don't tag people to review PR's. They will be reviewed when the team hasv time.

File: automatic/selenium-chromium-edge-driver/update.ps1 [hunk 1]
```diff
@@ -36,14 +34,14 @@ function global:au_SearchReplace {
 }
 
 function global:au_GetLatest {
-  # We use the available package on CCR as the source of truth
-  $packageVersion = Get-EdgePackageVersion -PackageName 'microsoft-edge'
-  $escapedVersion = [regex]::Escape($packageVersion)
+  $edgeDriverStableVersionURL = "https://msedgewebdriverstorage.blob.core.windows.net/edgewebdriver/LATEST_STABLE"
```
Comment (pauby) [inline automatic/selenium-chromium-edge-driver/update.ps1:37]: Please move this up to where `$releases` is on line 3 and call it that. These files have a format that we should follow.

File: automatic/selenium-chromium-edge-driver/update.ps1 [hunk 2]
```diff
@@ -36,14 +34,14 @@ function global:au_SearchReplace {
...
-  $downloadPage = Invoke-WebRequest $releases -UseBasicParsing
+  Invoke-WebRequest -URI $edgeDriverStableVersionURL -OutFile $stableVersionFile -UseBasicParsing
+  $packageVersion = Get-Content $stableVersionFile
 
-  $url32 = $downloadPage.Links | Where-Object { $_.href -match "$escapedVersion.*win32" } | Select-Object -First 1 -ExpandProperty href
-  $url64 = $downloadPage.Links | Where-Object { $_.href -match "$escapedVersion.*win64" } | Select-Object -First 1 -ExpandProperty href
+  $url32 = "https://msedgedriver.azureedge.net/$packageVersion/edgedriver_win32.zip"
+  $url64 = "https://msedgedriver.azureedge.net/$packageVersion/edgedriver_win64.zip"
 
   if (!$url32 -or !$url64) {
```
Comment (pauby) [inline automatic/selenium-chromium-edge-driver/update.ps1:46]: The code changes above makes this redundant as `$url32` and `$url64` will not be empty.

File: automatic/selenium-chromium-edge-driver/update.ps1 [hunk 3]
```diff
@@ -36,14 +34,14 @@ function global:au_SearchReplace {
 }
 
 function global:au_GetLatest {
-  # We use the available package on CCR as the source of truth
-  $packageVersion = Get-EdgePackageVersion -PackageName 'microsoft-edge'
```
Comment (pauby) [inline automatic/selenium-chromium-edge-driver/update.ps1:40]: As you're removing the call to this function, please remove the function itself.

**Review by pauby — CHANGES_REQUESTED** [review]

File: automatic/selenium-chromium-edge-driver/update.ps1 (hunk 3 above)
Comment (patest-dev) [reply automatic/selenium-chromium-edge-driver/update.ps1:40]: good point. I removed the function

File: automatic/selenium-chromium-edge-driver/update.ps1 (hunk 1 above)
Comment (patest-dev) [reply automatic/selenium-chromium-edge-driver/update.ps1]: Thx, makes sense. Didn't know the convention as this is my first PR here

File: automatic/selenium-chromium-edge-driver/update.ps1 (hunk 2 above)
Comment (patest-dev) [reply automatic/selenium-chromium-edge-driver/update.ps1]: Correct. Thx for the pointer

**Review by pauby — APPROVED** [review]

**Comment by pauby** [comment]
@patest-dev awesome work! Thanks for your contribution!

**Pull request merged**